### ISSUE TITLE
The lerna was not build all the packages as expected

### ISSUE THREAD

**Issue by blood-romantic** [lerna/lerna#3298]
## Current Behavior

- The lerna builds failed, if the `import { Header } from 'db-header';` was added to the top of the App.js that in db-main repo.
- The lerna builds success, if the `import { Header } from 'db-header';` was not added to the top of the App.js that in db-main repo.

[NOTE]: the `db-header` is a dependence of the `db-main`, and the `npx lerna bootstrap` has been executed.

## Expected Behavior

- The lerna builds success, if the `import { Header } from 'db-header';` was added to the top of the App.js that in db-main repo.
- The lerna builds success, if the `import { Header } from 'db-header';` was not added to the top of the App.js that in db-main repo.

## Steps to Reproduce

- add the dependence `import { Header } from 'db-header';` to the top of the App.js that in db-main repo.

- run the command:  `npx lerna run build`

The repo: https://github.com/blood-romantic/monorepo

### Failure Logs / Configuration

```txt

    ✔  db-header:build (3s)

    ✖  db-main:build
       > db-main@0.1.0 build
       > react-scripts build
       
       Creating an optimized production build...
       Failed to compile.
       
       Module not found: Error: Can't resolve 'db-header' in '/Users/gwang2/projects/Dashboard/packages/db-main/src'
```

<details><summary>lerna.json</summary><p>

```json
{
  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
  "useNx": true,
  "useWorkspaces": true,
  "version": "0.0.0"
}

```

</p></details>

<details><summary>lerna-debug.log</summary><p>

</p></details>

### Environment

lerna notice cli v5.4.3

 Environment info:

  System:
    OS: macOS 12.1
    CPU: (10) arm64 Apple M1 Pro
  Binaries:
    Node: 16.14.0 - /usr/local/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 8.3.1 - /usr/local/bin/npm
  Utilities:
    Git: 2.15.0 - /usr/local/bin/git
  npmPackages:
    lerna: ^5.4.3 => 5.4.3

**Comment by ghiscoding** [comment]
possible duplicate of #3289

**Comment by JamesHenry** [comment]
Hi @blood-romantic this should be resolved in 5.5.1 thanks to @fahslaj
